app/lifecycle/server.go

```diff
@@ -143,15 +147,12 @@ func SpawnServer(ctx context.Context, command string) (chan int, error) {
 			default:
 				crashCount++
 				slog.Warn(fmt.Sprintf("server crash %d - exit code %d - respawning", crashCount, code))
-				time.Sleep(500 * time.Millisecond)
-				if err := cmd.Start(); err != nil {
```

Comment:
`Start()` can only be called once for each `exec.Cmd` instance and so we need to create a new one on every restart